### DISCUSSION TITLE
White list vs KeepList and Grey vs Restart ?

### DISCUSSION THREAD

**Discussion by LaurentGrenet** [Cookie-AutoDelete/Cookie-AutoDelete#1420] [Q&A / Support] [answered]
Hi,

Documentation in wiki uses the wording "KeepList" and "RestartList".
On the other hand, CAD parameterization talks about "White List" and "Grey List"

Pls confirm that "White List" and "KeepList" are two wordings for the same (and are exactly equivalent)
and "Grey List" and "RestartList" also.

**Comment by kennethtran93** [comment, marked as answer]
Yes they are currently the same.  That was the intent behind the 'language clarification'.  However, editing the wiki is a separate 'repository', so I cannot have the wiki adjustments show up at the same time as when the PR/code changes.

**Comment by xzenor** [comment]
Thanks, I was about to put in an issue that there's no mention of what the difference between whitelist en greylist is. There's no mention of whitelist and greylist at all actually. I've been using CAD for a few months now and all this time it has been Whitelist en Greylist.   

Now I get that it might be a time issue (life and all) that the wiki is behind on the code but it's really confusing for newcomers.  

May I suggest putting a note somewhere on the wiki explaining that whitelist == keeplist and greylist == restartlist (in normal english of course) so that searching for either of those terms at least points a new user in the right direction.. That way there's no need to rewrite it all yet and still make it clear for newcomers.

**Reply by modernman1** [reply]
I have spent a LOT of time trying to understand what the heck the WHITELIST and GREYLIST options are on the interface - there is not a mention AT ALL of these words in the documentation/help, etc. - just not acceptable from a usability standpoint - 
(and the explanation of restartlist and keeplist and use of those terms is confusing as well.)

That said, I appreciate the work you are doing and making this extension available - and I appreciate that more than I dislike having had to go to great lengths to try and understand how to use it. So Thank You to to maintainer!
